src/common/types/stripe.ts

```
@@ -9,7 +9,8 @@ export const invoiceLinkPostRequestSchema = z.object({
   invoiceId: z.string().min(1),
   invoiceAmountUsd: z.number().min(50),
   contactName: z.string().min(1),
-  contactEmail: z.string().email()
+  contactEmail: z.string().email(),
+  achPaymentsEnabled: z.boolean()
```

ESLint

[error] Insert `,`

(prettier/prettier)